packages/multimedia/rtmpdump/package.mk

```diff
@@ -1,9 +1,10 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="rtmpdump"
-PKG_VERSION="fa8646daeb19dfd12c181f7d19de708d623704c0"
-PKG_SHA256="dba4d4d2e1c7de6884b01d98194b83cab6784669089fa3c919152087a3a38fd2"
+PKG_VERSION="c5f04a58fc2aeea6296ca7c44ee4734c18401aa3"
```

Comment:
please make sure inputstream.rtmp and curl build with this bump as rtmpdump has been problematic in the past.

Reply:
Confirmed both 
- https://curl.haxx.se/download/curl-7.74.0.tar.xz
- https://codeload.github.com/xbmc/inputstream.rtmp/tar.gz/3.4.0-Matrix

Building with
- HEAD c5f04a58fc2aeea6296ca7c44ee4734c18401aa3